### PR TITLE
fix(typing): botocore.get_kinesis_data_object

### PR DESCRIPTION
**Pull request by ronakts**
Fixing type error. https://github.com/DataDog/dd-trace-py/issues/3640

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: ddtrace/contrib/botocore/patch.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def inject_trace_to_eventbridge_detail(params, span):
 
 
 def get_kinesis_data_object(data):
-    # type: (str, Optional[bool]) -> Optional[Dict[str, Any]]
+    # type: (str) -> Optional[Dict[str, Any]]
     """
     :data: the data from a kinesis stream
     :try_b64: whether we should try to decode the string as base64
```
Comment (Kyle-Verhoog) [inline ddtrace/contrib/botocore/patch.py:167]:
```suggestion
```

should probably update this outdated line too

**Review by Kyle-Verhoog — APPROVED** [review]

File: ddtrace/contrib/botocore/patch.py (hunk 1 above)
Comment (ronakts) [reply ddtrace/contrib/botocore/patch.py]: done

**Review by Kyle-Verhoog — APPROVED** [review]
thanks @ronakts!

**Comment by ronakts** [comment]
@Kyle-Verhoog Is there anything I need to do before your team merge this PR?

**Comment by Kyle-Verhoog** [comment]
We're all good here @ronakts, we just need one more approval from the team and it's good to go! 😎

**Review by majorgreys — APPROVED** [review]

**Pull request merged**